### PR TITLE
Initial contribution of API spec location.yaml v0.1.0

### PR DESCRIPTION
**Pull request by akoshunyadi**
Initial contribution of the API spec location.yaml with the version 0.1.0. #4 
The API enables to verify the current position of the UE against given coordinates and accuracy.

### CODE REVIEW

**Review by jlurien — COMMENTED** [review]
To move this API forward, I suggest to merge this initial version as v0.1.0, and start a new dev branch to make contributions for v0.2.0. We can then move our comments there.

File: code/API_definitions/location.yaml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  version: 0.1.0
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+  - BasicAuth: []
+  - apiKey: []
+  - three_legged:
+    - device-location-read
```
Comment (jlurien) [inline code/API_definitions/location.yaml:20]:
```suggestion
    - device-location-verification-verify-read
```

File: code/API_definitions/location.yaml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location/v0
```
Comment (jlurien) [inline code/API_definitions/location.yaml:28]:
```suggestion
        default: device-location-verification/v0
```

File: code/API_definitions/location.yaml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+        description: Base path for the device location API
```
Comment (jlurien) [inline code/API_definitions/location.yaml:29]:
```suggestion
        description: Base path for the device location verification API
```

File: code/API_definitions/location.yaml [hunk 4]
```diff
@@ -0,0 +1,253 @@
+openapi: 3.0.3
+info:
+  title: Device location API
```
Comment (jlurien) [inline code/API_definitions/location.yaml:3]:
```suggestion
  title: Device Location Verification API
```

**Review by jlurien — COMMENTED** [review]
Change name more specifically to "Device Location Verification"

**Review by jlurien — APPROVED** [review]

**Comment by shilpa-padgaonkar** [comment]
As agreed, merging the pull request so that we can have a base version. New issues should be created to bring in further improvements in the spec.

**Pull request merged**